rero_ils/modules/loans/api.py

```diff
@@ -143,7 +143,7 @@ def is_notified(self, notification_type=None):
 
     def create_notification(self, notification_type=None):
         """Creates a recall notification from a checked-out loan."""
-        data = {}
+        notification = {}
```

Comment:
Why not `notification = None`?

Reply:
This code is from Aly, I dont know.

Reply:
Cause we are returning a dict.  I can change it to None if necessary.